include/cocaine/context.hpp

```diff
@@ -152,6 +169,9 @@ class context_t {
     // because services are allowed to start and stop other services during their lifetime.
     synchronized<service_list_t> m_services;
 
+    //Flag set after initialization
+    bool ready;
```

Comment:
As I see from docs - there is no such possibility right out of the box - but I think I can try to implement one. It will get some time though.